### PR TITLE
[5-METHODS] Recap methods

### PR DESCRIPTION
**Pull request by olayinkasf**

### CODE REVIEW

File: lesson5/index.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+title: Recap Functions & Methods
+nav_order: 5
+has_children: true
+---
+
+# Lesson 4: Recap Functions & Methods
+
+## Goals
+* Recap method `signature`, `arguments` and `return` value
+* Recap `recursion`
```
Comment (fensterzumberg) [inline lesson5/index.md:11]: We should add a link to this: https://www.google.com/search?source=hp&ei=m3ZXX8CVHMrWatmLg6gJ&q=recursion&oq=recursion&gs_lcp=CgZwc3ktYWIQAzIFCAAQsQMyAggAMgIIADICCAAyAggAMgIIADICCAAyAggAMgIIADICCAA6CAgAELEDEIMBOgUILhCxAzoKCAAQsQMQRhD_AVC_B1iCEWCJE2gBcAB4AIABU4gBxgSSAQE5mAEAoAEBqgEHZ3dzLXdperABAA&sclient=psy-ab&ved=0ahUKEwiAz7asxdnrAhVKqxoKHdnFAJUQ4dUDCAk&uact=5 :-)

File: lesson5/index.md [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+      //code to execute
+    }
+}
+```
+
+Here, we have created a method named `main`. 
+We can see that we have used the `public`, `static` and `void` before the method name.
+
+- `public` - access modifier. It means the method can be accessed from anywhere.
+- `static` here means this method belongs to the class Main and not to a specific instance of Main.
```
Comment (fensterzumberg) [inline lesson5/index.md:36]: potentially will cause confusion - but as long as we're aware and anticipating this it's fine to add I think

Comment (olayinkasf) [reply lesson5/index.md:36]: Yeah I plan to mention it briefly with a tiny example. But the scope is still writing `static` methods.

File: lesson5/index.md [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+And sometimes it is less subtle and usually involves multiple methods. For example:
+
+```java
+public class Main {
+
+    public int foo(int a) {
+        return bar(a);
+    }
+
+    public int subtract(int a) {
```
Comment (fensterzumberg) [inline lesson5/index.md:182]: this should be bar(int a)

File: lesson5/index.md [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+
+- Write a function that takes three numbers and returns the minimum number of the three
+- Write another function that takes three numbers and returns the maximum number of the three
+
+### 5 - Recursion: Logarithm
+
+Using recursion, write a function that given an integer, it returns the number of digits representing that integer
+in base 10 e.g. `f(141) = 3` because `141` has three digits.
+
+### 6 - Recursion: Power
```
Comment (fensterzumberg) [inline lesson5/index.md:244]: Should be Recursion: Fibonacci, right?

File: lesson5/index.md (hunk 1 above)
Comment (olayinkasf) [reply lesson5/index.md:11]: I added it to the header of the recursion section.

**Review by fensterzumberg — APPROVED** [review]

**Pull request merged**